client/src/components/dashboard/Dashboard.js

```diff
@@ -1,23 +1,46 @@
 /* eslint-disable react/prop-types */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
+import axios from 'axios';
 import auth from '../../auth/auth';
 
-const Dashboard = props => (
-  <>
-    <div>Dashboard</div>
-    <Link to="/">Home</Link>
-    <button
-      type="button"
-      onClick={() => {
-        auth.logout(() => {
-          props.history.push('/login');
-        });
-      }}
-    >
-Logout
-    </button>
-  </>
-);
+const Dashboard = (props) => {
+  const [userName, setUserName] = useState('');
+
+  useEffect(() => {
+    axios.get(process.env.REACT_APP_DASHBOARD_URL, {
+      headers: {
+        'x-auth-token': localStorage.getItem('jwt'),
```

Comment:
We also need a layer to get data from. Although localstorage is accessible to all components, a fine layer of abstraction is always better, you can even build a HOC that passes this component required data. But it's not big of a deal now.